capitulo 1/lang_en.json

```diff
@@ -5904,65 +5904,65 @@
   "scr_text_slash_scr_text_gml_3098_0": "\\C4 ",
   "scr_text_slash_scr_text_gml_2965_0": " #Sim",
   "scr_text_slash_scr_text_gml_2636_0": " #Não",
-  "scr_text_slash_scr_text_gml_774_0": "\\M0* Ei..^1. olá..^1.&* Vi que você tava..^1.&* Aqui.../",
-  "scr_text_slash_scr_text_gml_775_0": "\\M1* Sou eu^1!&* Você ME conhece^1, certo^1?&* É claro que sim!/",
+  "scr_text_slash_scr_text_gml_774_0": "\\M0* Oi..^1. ai..^1.&* Vi que você tava..^1.&* Aqui.../",
+  "scr_text_slash_scr_text_gml_775_0": "\\M1* Sou eu^1!&* Você ME conhece^1, certo^1?&* É claro que conhece!/",
   "scr_text_slash_scr_text_gml_776_0": "\\M0* Eu sou MUITO popular^1!&* Eu só!/",
   "scr_text_slash_scr_text_gml_777_0": "\\M3* Não tenho amigos.../",
-  "scr_text_slash_scr_text_gml_778_0": "\\M1* Mas tá tudo BEM^1, ouviu^1!&*Porque você está aqui!/",
+  "scr_text_slash_scr_text_gml_778_0": "\\M1* Mas tá tudo BEM^1, ouviu^1!&* Porque você tá aqui!/",
   "scr_text_slash_scr_text_gml_779_0": "\\M0* VOCÊ vai ser meu amigo?/",
   "scr_text_slash_scr_text_gml_2714_0": "\\C2 ",
   "scr_text_slash_scr_text_gml_786_0": " #Kris",
   "scr_text_slash_scr_text_gml_787_0": " #Hipo-#pótamo",
-  "scr_text_slash_scr_text_gml_794_0": "\\M4* Ah^1! Ah^1!&* Eu tô tão feliz^1, ouviu!^1?&* Eu estou tão feliz que Eu..^1. Eu.../",
+  "scr_text_slash_scr_text_gml_794_0": "\\M4* Ah^1! Ah^1!&* Eu tô tão feliz^1, ouviu!^1?&* Eu estou tão feliz que eu../",
   "scr_text_slash_scr_text_gml_795_0": "\\M3* Eu não sei o que fazer^1!&* Eu nunca estive tão feliz assim antes!/",
   "scr_text_slash_scr_text_gml_796_0": "\\M2* Qual seu nome^1, amigo?/",
   "scr_text_slash_scr_text_gml_2751_0": "\\C2 ",
-  "scr_text_slash_scr_text_gml_809_0": " #Cebola",
-  "scr_text_slash_scr_text_gml_810_0": " #Bonito",
+  "scr_text_slash_scr_text_gml_809_0": " #Cebolinha",
+  "scr_text_slash_scr_text_gml_810_0": " #Bonitinho",
   "scr_text_slash_scr_text_gml_811_0": "Asriel II",
   "scr_text_slash_scr_text_gml_812_0": "Repulsivo",
-  "scr_text_slash_scr_text_gml_817_0": "\\M0* Kris^1! É você^1!&* Esse é um nome bem legal pra um amigo!/",
-  "scr_text_slash_scr_text_gml_822_0": "\\M0* H..^1. Hipo..^1. Hiper..^1. Horta..^1.&* Um^1, Hipopótamo^1! É você^1!&* Grande nome^1, mas que digno!/",
+  "scr_text_slash_scr_text_gml_817_0": "\\M0* Kris^1! OK^1!&* Esse é um nome bem legal pra um amigo!/",
+  "scr_text_slash_scr_text_gml_822_0": "\\M0* H..^1. Hipo..^1. Hiper..^1. Horta..^1.&* Um^1, Hipopótamo^1! OK^1!&* Grande nome^1, mas vale a pena!/",
```

Comment:
Mesma situação do anterior, mas se nesse é pra evitar o problema da quebra de linha pode manter, senão sugiro "Certo!".
Sobre a segunda alteração, que fica em "...but it's worth it", eu interpreto worth como um adjetivo ao nome, pois dizer que um nome vale a pena soa estranho. Por isso, preferi algo que indique que acha que é um nome digno (a altura) da pessoa com quem ele está conversando.  Ref.: https://www.linguee.com.br/ingles-portugues/traducao/worth.html